consensus/spos/commonSubround/subroundBlock.go

```diff
@@ -361,6 +367,17 @@ func (sr *SubroundBlock) processReceivedBlock(cnsDta *consensus.Message) bool {
 	return true
 }
 
+func (sr *SubroundBlock) computeSubroundProcessingMetric(startTime time.Time, metric string) {
+	subRoundDuration := sr.EndTime() - sr.StartTime()
```

Comment:
I would delete lines 371-375 and I would use in line 377 sr.Rounder().TimeDuration() instead subRoundDuration, as processing time could reached to 85% from round duration time and you could have a result value greater than 100 in percent variable

Reply:
As discussed, this compute the percentage against the block subround, not the whole round